app/angular/package.json

```diff
@@ -73,6 +73,7 @@
     "@angular/forms": "^11.0.0",
     "@angular/platform-browser": "^11.0.0",
     "@angular/platform-browser-dynamic": "^11.0.0",
+    "@nrwl/workspace": ">=11.1.0",
```

Comment:
Ok, I'll use `11.1.0-beta.2`! How can I capture the yarn.lock changes, though?

Reply:
It still defaults to 
```"
@nrwl/workspace@>=11.1.0-beta.2":
  version "9999.0.0"
```
since `11.1.0-beta.2` is not an actual tag/release. We can wait until the end of the week, when that release will happen, maybe?

Reply:
Yes sure, I let you update that when the release will be done and then ping me back 😉

Reply:
@gaetanmaisse I committed the `yarn.lock` changes too!